doc/api_assets/style.css

```diff
@@ -373,6 +373,11 @@ p {
   padding-bottom: 2rem;
 }
 
+/* prevent the module-level sticky stability header from overlapping the section headers when clicked */
+#apicontent:has(> .api_stability) a {
```

Comment:
this means that every single `a` of every single page that has an api stability banner will receive the scroll-margin-top attribute.... is that correct? cause the comments suggests to me that this is only specific for headers?